package.json

```diff
@@ -54,7 +54,9 @@
   },
   "homepage": "https://github.com/danny-avila/LibreChat#readme",
   "dependencies": {
-    "axios": "^1.4.0"
+    "axios": "^1.4.0",
+    "nodemon": "3.0.1",
+    "passport-twitter": "^0.1.5"
```

Comment:
nodemon and passport-twitter need to be uninstalled from the root directory and instead installed in the api directory